packages/config/config/devices/0x019b/zm_single_relay_16.json

```diff
@@ -0,0 +1,228 @@
+{
+	"manufacturer": "HeatIt",
+	"manufacturerId": "0x019b",
+	"label": "ZM SINGLE RELAY 16",
+	"description": "Heatit ZM Single Relay 16A",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x3500",
+			"zwaveAllianceId": 4062
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External Relay Control S1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Control External Start/Stop S2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "External Relay Control S2",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Control External Start/Stop S2",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Load Limit",
+			"description": "1-16 A",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 16,
+			"defaultValue": 16,
+			"unsigned": true
+		},
+		"2": {
+			"label": "Power Shutdown Actions",
+			"description": "0: Disabled, 1-32767: minute(s) delay before attempting to turn back on",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 20,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled and will not retry. User needs to manually turn on afterwards. If temperature overload is on, device will not turn on until device has cooled down",
+					"value": 0
+				}
+			]
+		},
+		"3": {
+			"label": "Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "S1/Button Operation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Button turns load on/off and sends meter report + relay status",
+					"value": 0
+				},
+				{
+					"label": "Button sends meter report + relay status, load can be only controlled wirelessly",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Scene Notifications",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Sends scene controller for s2. s1 disabled",
```

Comment:
What's "scene controller"? Central Scene Notifications? In that case something like this would probably be better:
```
"label": "Central Scene Notifications",
"description": "Configure which buttons cause notifications to be sent",
```
with these options:
* S2 only
* S1 only
* S1 & S2
* None